collector/infiniband_linux.go

```diff
@@ -148,6 +148,11 @@ func readMetric(directory, metricFile string) (uint64, error) {
 		return 0, err
 	}
 
+	switch metricFile {
+	case "port_rcv_data", "port_xmit_data", "port_rcv_data_64", "port_xmit_data_64":
```

Comment:
Maybe put a comment here about why we're doing this, either link to docs, or link to issue.